rnd/autogpt_builder/src/components/edit/control/BlocksControl.tsx

```diff
@@ -16,9 +16,11 @@ import { PlusIcon } from "@radix-ui/react-icons";
 import { IconToyBrick } from "@/components/ui/icons";
 import SchemaTooltip from "@/components/SchemaTooltip";
 import { getPrimaryCategoryColor } from "@/lib/utils";
+
 interface BlocksControlProps {
   blocks: Block[];
   addBlock: (id: string, name: string) => void;
+  pinBlocks: boolean;
```

Comment:
Nam this more clearly

Reply:
`pinBlocksPopover`, `isBlocksPopoverPinned`, `showBlocksPopover`